croquemort/crawler.py

```diff
@@ -11,6 +11,12 @@
 
 HEAD_TIMEOUT = 10  # in seconds
 GET_TIMEOUT = 3 * 60  # in seconds
+# domains that won't answer correctly to HEAD requests
+# TODO move to config file
+KNOWN_HEAD_OFFENDER_DOMAINS = [
```

Comment:
:+1: but would be awesome to have this externalisable (ie. not an hardcoded list but a configurable entry) so we may keep this up to date without having to perform a release (just add the new domains to ansible vars and just redeploy the conf)

Reply:
Yep, like I said above I will put this in a config file in v2 which should arrive very shortly. I made some changes to the way the config is handled in dev and it's simpler if I make the change there.

Reply:
:+1: